### DISCUSSION TITLE
how does an nvshad update work?

### DISCUSSION THREAD

**Discussion by Manix80** [NvChad/NvChad#2838] [Q&A] [answered]
I've modified some files quite a bit, such as "chadrc.lua, mappings.lua".

if an update is to take place (for example a git clone) should I reconfigure everything?

**Comment by siduck** [comment, marked as answer]
@Manix80 no, chadrc values get merged by nvconfig.lua ( which is  in main nvchad repo ) so rm -rf .git in starter repo when u clone it! because its just an example on how to use nvchad as a plugin, it   basically pulls nvchad/nvchad repo as a plugin

**Reply by Manix80** [reply]
haaa yes, I just saw NvChad in lazy.nvim ...
sorry for the silly question
